apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -117,50 +117,51 @@ const DanceAiModal: React.FunctionComponent<DanceAiProps> = ({onClose}) => {
         inputLibrary.items.find((item: AiModalItem) => item.id === input).name
     );
     const request = `${promptString} ${inputNames.join(', ')}.`;
-    startAi(request);
+    startAi(inputs, request);
     setMode(Mode.GENERATING);
   };
 
-  const startAi = async (value: string) => {
-    const responseJsonString = await doAi(value);
-    const response = JSON.parse(responseJsonString);
+  const startAi = async (inputs: string[], value: string) => {
+    const resultJsonString = await doAi(value);
+    const result = JSON.parse(resultJsonString);
 
     // "Pick" a subset of fields to be used.  Specifically, we exclude the
     // explanation, since we don't want it becoming part of the code.
-    const pickedResponse = (({
+    const pickedResult = (({
       backgroundEffect,
       backgroundColor,
       foregroundEffect,
-      dancers,
-    }) => ({backgroundEffect, backgroundColor, foregroundEffect, dancers}))(
-      response
-    );
-    const pickedResponseJson = JSON.stringify(pickedResponse);
+    }) => ({
+      backgroundEffect,
+      backgroundColor,
+      foregroundEffect,
+    }))(result);
+
+    const fullResult = {inputs, ...result};
```

Comment:
Do you intend `fullResult = {inputs, ...pickedResult};` ?

Reply:
Yes, thanks!